apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -96,9 +96,17 @@ const readMeterReportingPeriods = async ({ context, organizations }) => {
     })
 }
 
-const checkIsDateStartOrEndOfPeriod = (date, today, start, end) => {
-    return dayjs(date).format('YYYY-MM-DD') === dayjs(today).set('date', start).format('YYYY-MM-DD') ||
-        dayjs(date).format('YYYY-MM-DD') === dayjs(today).set('date', end).format('YYYY-MM-DD')
+const checkIsDateStartOrEndOfPeriod = (date, today, start, end) => (
+    dayjs(date).format('YYYY-MM-DD') === dayjs(today).set('date', start).format('YYYY-MM-DD') ||
```

Comment:
You don't need to pass `today` argument, because you can create it inside the function.
Also, you can compare dayjs objects using `isSame` method with specified comparison granularity set to `'day'`